Moving Platform and Scratch Sprites

Vikingur opened the pull request:
5 second interval, very primitive directionality.

Vikingur committed:
Moving Platform and Scratch Sprites

5 second interval, very primitive directionality

ashleycrouch commented:
You should make the cycle_time and the platform_body public, and then this can be brought to the main branch.

Vikingur committed:
Changed access on requested vars

platform_body and cycle_time are now public and can be adjusted by
outside scripts.

Vikingur commented:
Done. Do I have to do another pull request or are the changes still considered to be a part of this one?

ashleycrouch committed:
Moved platform with script to prefabs

ashleycrouch committed:
updateFromMaster

update
